launchpod/angular-launchpod/src/app/transcribe-form/transcribe-form.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit } from '@angular/core';
+import { FormHandlerService } from '../form-handler.service';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { HttpParams } from '@angular/common/http';
+
+@Component({
+  selector: 'app-transcribe-form',
+  templateUrl: './transcribe-form.component.html',
+  styleUrls: ['./transcribe-form.component.css']
+})
+export class TranscribeFormComponent implements OnInit {
+
+  feedValue: string;
+
+  constructor(private formHandlerService: FormHandlerService) { }
+
+  ngOnInit(): void {
+  }
```

Comment:
Same here with the nits above. If I'm missing something please let me know.

Reply:
done.